samples/bluetooth/mesh/light/README.rst

```diff
@@ -62,9 +66,12 @@ The following table shows the mesh light composition data for this sample:
    Gen. OnOff Server
    =================  =================  =================  =================
 
+.. note::
+   When used with :ref:`zephyr:thingy53_nrf5340`, Element 4 is not available.
```

Comment:
Maybe inform that Thingy:53 supports only one RGB LEDs.
I would also state that RGB channels are used as "separate LEDs" (we mentioned that in other samples already).